tests/behat/features/api/get_scratch_remix_programs.feature

```diff
@@ -0,0 +1,123 @@
+    And the uploaded program should have a Scratch parent having id "70058680"
+    And the uploaded program should have no further Scratch parents
+    And the uploaded program should have no Catrobat forward descendants except self-relation
+    And the uploaded program should have RemixOf "Music Inventor [https://scratch.mit.edu/projects/29495624], The Colour Divide - Trailer [https://scratch.mit.edu/projects/70058680/]" in the xml
+    Given I have a parameter "limit" with value "6"
+    And I have a parameter "offset" with value "0"
+    When I GET "/app/api/projects/scratchRemixes.json" with these parameters
+    Then I should get 6 programs in the following order:
+      | Name       |
+      | program 5  |
+      | program 6  |
+      | program 7  |
+      | program 8  |
+      | program 9  |
+      | program 18 |
+
+  Scenario: program upload with parent-URL referring to existing Catrobat programs and
+  Catrobat language version 1.0 should correctly add remix relations (example #2)
+    Given there are Scratch remix relations:
+      | scratch_parent_id | catrobat_child_id |
+      | 29495624          | 10                |
+      | 70058680          | 11                |
+
+    And there are backward remix relations:
+      | parent_id | child_id |
+      | 9         | 8        |
+      | 6         | 4        |
+      | 6         | 1        |
+
+    Given I have a program with "catrobatLanguageVersion" set to "1.0" and "url" set to "The Colour Divide - Trailer [https://scratch.mit.edu/projects/70058680/], Merge 2 [Program 2 [/pocketalice/project/2], Merge 1 [Program 6 [/app/project/6], Program 8 [https://share.catrob.at/app/project/8]]]"
+    When I upload the program with the id "18" and name "program 18"
+    And the uploaded program should have remix migration date NOT NULL
+    And the uploaded program should have a Catrobat forward ancestor having its own id and depth "0"
+    And the uploaded program should have no Catrobat backward parents
+    And the uploaded program should have a Scratch parent having id "70058680"
+    And the uploaded program should have no further Scratch parents
+    And the uploaded program should have no Catrobat forward descendants except self-relation
+    And the uploaded program should have RemixOf "The Colour Divide - Trailer [https://scratch.mit.edu/projects/70058680/], Merge 2 [Program 2 [/pocketalice/project/2], Merge 1 [Program 6 [/app/project/6], Program 8 [https://share.catrob.at/app/project/8]]]" in the xml
+    Given I have a parameter "limit" with value "9"
+    And I have a parameter "offset" with value "0"
+    When I GET "/app/api/projects/scratchRemixes.json" with these parameters
+    Then I should get 9 programs in the following order:
+      | Name       |
+      | program 5  |
+      | program 6  |
+      | program 7  |
+      | program 8  |
+      | program 9  |
+      | program 10 |
+      | program 11 |
+      | program 18 |
+      | program 12 |
```

Comment:
![image](https://user-images.githubusercontent.com/40868718/67566063-27800f80-f727-11e9-845e-e52a419af7cc.png)

This test does not fail but it definitely should. Fix the glue code and test.

As a side note. Try to first write your failing tests and then start implementing the code. At last you can check if all your tests are passing ;)
![image](https://user-images.githubusercontent.com/40868718/67566304-bb51db80-f727-11e9-8364-fb0935ad9f12.png)